### ISSUE TITLE
PR Comment uses ID instead of friendly name

### ISSUE THREAD

**Issue by chris3ware** [aquasecurity/tfsec-pr-commenter-action#49]
Hi, I just wondered if this was expected behaviour? The vscode extension / cli highlights the following code with the message:

```hcl
variable "env_analytics_token" {
  description = "Google Analytics token"
  type        = string
  default     = "token"
}
```

```zsh
Result #1 CRITICAL Variable includes a potentially sensitive default value. 
───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
 services/frontend-app/variables.tf Line 176
───────┬───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
  173  │ variable "env_analytics_token" {
  174  │   description = "Google Analytics token"
  175  │   type        = string
  176  │   default     = "token"
  177  │ }
───────┴───────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
          **ID general-secrets-no-plaintext-exposure**
```

However the PR commenter returns the ID instead, as shown below:

<img width="858" alt="Screenshot 2022-02-03 at 13 09 34" src="https://user-images.githubusercontent.com/36608309/152349921-dd685fc3-ddf8-4d49-9681-baf01018a4dd.png">

**Comment by owenrumney** [comment]
It should be more consistent, will update the commenter to use the longID

Thanks for highlighting

**Comment by owenrumney** [comment]
Hey @chris3ware  - this is resolved in v1.0.4 or @main

Thanks again for the alert
